### ISSUE TITLE
'make install' fails when '-DBUILD_TOOLS=OFF' is used

### ISSUE THREAD

**Issue by rubadub** [Electrostatics/apbs-pdb2pqr#467]
When using `-DBUILD_TOOLS=OFF` with `cmake` the subsequent `make install` fails with this error:

```
CMake Error at cmake_install.cmake:52 (FILE):
  file INSTALL cannot find
  "/export1/Builds/SandBox-2016.10.11-14.05/apbs-pdb2pqr/apbs/b/tools/bin".
make: *** [install] Error 1
```

I guess `CMakeLists.txt` needs to be updated for the case when building of tools is turned off.

**Comment by kozlac** [comment]
@rubadub can you tell me your setup and the steps for building? I successfully built it in Windows 7 with VS14 and Ubuntu14.04 GCC4.8 with tools off.

**Comment by rubadub** [comment]
This is on CentOS 7.2 with GCC 4.8.5. I don't think the OS or CC matters, the tools directory is in the CMakeLists.txt install target regardless -DBUILD_TOOLS setting.

**Comment by kozlac** [comment]
Solved with commit d6bc46a.
